### ISSUE TITLE
parent-placeholder displayed in the html

### ISSUE THREAD

**Issue by wingsline** [laravel/framework#17523]
- Laravel Version: 5.4.0
- PHP Version: 7.0.14
- Database Driver & Version: -

### Description:

Using the layout example in the docs (https://laravel.com/docs/5.4/blade#extending-a-layout) and the child example blade template, when viewed in the browser the "parent-placeholder" gets displayed.

When the template is rendered the first time, it displays properly, after that it displays the 'parent-placeholder'. 

### Steps To Reproduce:

Fresh application created using `laravel new blog` command.

Use the https://laravel.com/docs/5.4/blade#defining-a-layout layout

Use the https://laravel.com/docs/5.4/blade#extending-a-layout template

Result:

```
<html>
    <head>
        <title>App Name - Page Title</title>
    </head>
    <body>
            ##parent-placeholder-19bd1503d9bad449304cc6b4e977b74bac6cc771##

    <p>This is appended to the master sidebar.</p>

        <div class="container">
                <p>This is my body content.</p>
        </div>
    </body>
</html>
```

**Comment by sisve** [comment]
This looks like it was fixed in 5.4.1 - https://github.com/laravel/framework/commit/64f7e9c4e37637df7b0820b11d5fcee1c1cca58d

**Comment by themsaid** [comment]
yeah please upgrade to 5.4.1
